kroxylicious-filters/kroxylicious-record-validation/src/test/java/io/kroxylicious/proxy/filter/schema/JsonSchemaBytebufValidatorTest.java

```diff
@@ -0,0 +1,157 @@
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import com.github.tomakehurst.wiremock.WireMockServer;
+import com.github.tomakehurst.wiremock.client.WireMock;
+
+import io.kroxylicious.proxy.filter.schema.validation.Result;
+import io.kroxylicious.proxy.filter.schema.validation.bytebuf.BytebufValidator;
+import io.kroxylicious.proxy.filter.schema.validation.bytebuf.BytebufValidators;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.get;
+import static com.github.tomakehurst.wiremock.client.WireMock.urlEqualTo;
+import static com.github.tomakehurst.wiremock.core.WireMockConfiguration.wireMockConfig;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verifyNoMoreInteractions;
+
+public class JsonSchemaBytebufValidatorTest {
+
+    static WireMockServer registryServer;
+
+    private static final String JSON_SCHEMA = """
+            {
+              "$id": "https://example.com/person.schema.json",
+              "$schema": "http://json-schema.org/draft-07/schema#",
+              "title": "Person",
+              "type": "object",
+              "properties": {
+                "firstName": {
+                  "type": "string",
+                  "description": "The person's first name."
+                },
+                "lastName": {
+                  "type": "string",
+                  "description": "The person's last name."
+                },
+                "age": {
+                  "description": "Age in years which must be equal to or greater than zero.",
+                  "type": "integer",
+                  "minimum": 0
+                }
+              }
+            }
+            """;
+
+    @BeforeAll
+    public static void initMockRegistry() {
+        registryServer = new WireMockServer(
+                wireMockConfig()
+                        .dynamicPort());
+
+        registryServer.start();
+
+        registryServer.stubFor(
+                get(urlEqualTo("/apis/registry/v2/ids/globalIds/1?dereference=false"))
+                        .willReturn(WireMock.aResponse()
+                                .withHeader("Content-Type", "application/json")
+                                .withBody(JSON_SCHEMA)));
+
+        registryServer.stubFor(
+                get(urlEqualTo("/apis/registry/v2/ids/globalIds/1/references"))
+                        .willReturn(WireMock.aResponse()
+                                .withHeader("Content-Type", "application/json")
+                                .withBody("[]")));
+    }
+
+    @AfterAll
+    public static void shutdownMockRegistry() {
+        registryServer.shutdown();
+    }
+
+    @Test
+    void testValueValidated() {
+        BytebufValidator mockValidator = mock(BytebufValidator.class);
```

Comment:
why is `mockValidator` here?  its not passed into the `jsonSchemaValidator` or the call to `validate`

Reply:
Removed, it's not needed.